addons/mail/static/src/components/welcome_view/welcome_view.js

```diff
@@ -22,6 +22,9 @@ export class WelcomeView extends owl.Component {
         return this.messaging && this.messaging.models['mail.welcome_view'].get(this.props.localId);
     }
 
+    get loggedInAsText() {
+        return _.str.sprintf(this.env._t("Logged in as %s"), this.messaging.currentUser.nameOrDisplayName);
```

Comment:
You may want to import `sprintf` from `@web/core/utils/strings` rather than relying on the one from `_`, which is removed in subsequent versions of Odoo.

The same goes for `_t`; I'd recommend you use the one from `@web/core/l10n/translation`

Reply:
done